src/main/java/bio/terra/stairway/FlightDebugInfo.java

```diff
@@ -12,6 +12,9 @@
 
   private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Suggest you add
```
.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
```
That will allow us to add optional properties and have them ignored by previous versions. As it should have been...

Reply:
Done.
I didn't realize that FAIL_ON_UNKNOWN_PRIMITIVES was default false. So now I think v0 serialized FlightDebugInfo will be readable by this new v1 reader, but v1 written FlightDebugInfo will not be readable by v0 readers due to FAIL_ON_UNKNOWN_PROPERTIES. Updating PR description accordingly.
I think this is ok to roll out because we don't use FlightDebugInfo outside of testing, right?

Reply:
Right on both counts.